lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -43,67 +58,174 @@ class UsesHTTP2Audit extends Audit {
     return {
       id: 'uses-http2',
       title: str_(UIStrings.title),
-      failureTitle: str_(UIStrings.failureTitle),
       description: str_(UIStrings.description),
-      requiredArtifacts: ['URL', 'devtoolsLogs'],
+      requiredArtifacts: ['URL', 'devtoolsLogs', 'traces'],
     };
   }
 
+  /**
+   * Computes the estimated effect all results being converted to use http/2, the max of:
+   *
+   * - end time of the last long task in the provided graph
+   * - end time of the last node in the graph
+   *
+   * @param {Array<{url: string}>} results
+   * @param {Node} graph
+   * @param {Simulator} simulator
+   * @return {number}
+   */
+  static computeWasteWithTTIGraph(results, graph, simulator) {
```

Comment:
Is there enough in common here and ByteEfficienyAudit to make a function that takes a graph transform cb and computes savings?

Reply:
I don't think so personally. It's distinct enough IMO that sharing will create more questions than the ability to see directly how savings is being computed. If we end up moving further in this direction away from byte efficiency then I agree it's worth exploring a refactor.